opentracing-spring-jaeger-starter/src/main/java/io/opentracing/contrib/java/spring/jaeger/starter/log/slf4j/ScopeManagerSlf4jDecorator.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2018-2019 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.java.spring.jaeger.starter.log.slf4j;
+
+import io.jaegertracing.internal.JaegerSpanContext;
+import io.opentracing.Scope;
+import io.opentracing.ScopeManager;
+import io.opentracing.Span;
+import io.opentracing.contrib.java.spring.jaeger.starter.JaegerConfigurationProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.MDC;
+
+
+/**
+ * @author <a href="mailto:taylor.tian@ericsson.com">Taylor Tian</a>
+ */
+public class ScopeManagerSlf4jDecorator implements ScopeManager {
+  private static final String B3_TRACE_ID = "X-B3-TraceId";
+  private static final String B3_SPAN_ID = "X-B3-SpanId";
+  private static final String B3_SAMPLED = "X-B3-Sampled";
+  private static final String B3_EXPORT = "X-Span-Export";
+  private static final String OPENTRACING_TRACE_ID = "ToTraceId";

```

Comment:
is the `ToTraceId` redered in logs as key for trace id? If so change it to `traceId`. The same for span id

Reply:
yes. I refered to opentracing specification [here](https://github.com/opentracing/specification/blob/master/rfc/trace_identifiers.md). It defines these two ID as ToTraceId & ToSpanId.

Reply:
Those are method accessor. For the field names I would prefer `traceId` or `trace-id` (w3c trace-context)

Reply:
Oh, I'll change them.